ql/src/java/org/apache/hadoop/hive/ql/ddl/database/create/CreateDatabaseAnalyzer.java

```diff
@@ -70,19 +73,43 @@ public void analyzeInternal(ASTNode root) throws SemanticException {
         managedLocationUri = unescapeSQLString(childNode.getChild(0).getText());
         outputs.add(toWriteEntity(managedLocationUri));
         break;
+      case HiveParser.TOK_DATACONNECTOR:
+        type = "REMOTE";
+        // locationUri = "REMOTE_DATABASE"; // TODO
+        ASTNode nextNode = (ASTNode) root.getChild(i);
+        connectorName = ((ASTNode)nextNode).getChild(0).getText();
+        outputs.add(toWriteEntity(connectorName));
+        // outputs.remove(toWriteEntity(locationUri));
```

Comment:
This commented LOC has also been deleted locally. 
Essentially, for a DB of type REMOTE, we do not want to honor any location or managedlocation values that could have been used in the DDL statement (although it might be valid syntactically, its incorrect semantics). So I remove them both.